packages/pharos/scripts/build-tokens.mjs

```diff
@@ -114,6 +114,22 @@ StyleDictionary.registerFormat({
   },
 });
 
+StyleDictionary.registerFormat({
+  name: 'javascript/es6-default',
+  formatter: function (dictionary) {
+    return (
+      fileHeader() +
+      dictionary.allProperties
+        .map(function (prop) {
+          var to_ret_prop = 'export default ' + JSON.stringify(prop.value) + ';';
+          if (prop.comment) to_ret_prop = to_ret_prop.concat(' // ' + prop.comment);
+          return to_ret_prop;
```

Comment:
```suggestion
          const export = `export default ${JSON.stringify(prop.value)};`;
          const comment = prop.comment ? ` // ${prop.comment}` : '';
          return `${export}{comment}`;
```

Reply:
`export` is a keyword 😛 Will refactor accordingly 👍

Reply:
I'm aware, that was a typo. Fixing my suggestion so you can commit it if you like.